Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Xamarin.Essentials
+{
+    public class MapLaunchOptions
+    {
+        public MapNavigation MapNavigation { get; set; }
+
+        public string Name { get; set; }
```

Comment:
Not sure if we should add name here? do we need it?

Reply:
Yeah we do, it is the title of the place we are launching. If you were to show a route to some location, you would also want to give it a name instead of just the coordinates